src/main/java/bio/terra/janitor/service/cleanup/StairwayLoggingHooks.java

```diff
@@ -0,0 +1,74 @@
+package bio.terra.janitor.service.cleanup;
+
+import bio.terra.janitor.db.TrackedResourceId;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.HookAction;
+import bio.terra.stairway.StairwayHook;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+public class StairwayLoggingHooks implements StairwayHook {
+  private static final String FlightLogFormat =
```

Comment:
nit: `private static final` constant variables should be `CAPITAL_CASE`
ditto others

Reply:
Done